inputs/simple_inner.json

```diff
@@ -29,6 +29,15 @@
         "image_id": "BUILT_IMAGE_ID"
       },
       "name": "all_rpm_packages"
+    },
+    {
+      "args": {
+        "imagestream": "{{IMAGESTREAM}}",
+        "docker_image_repo": "{{DOCKER_IMAGE_REPO}}",
+        "url": "{{OPENSHIFT_URI}}",
+        "verify_ssl": false
+      },
+      "name": "import_image"
```

Comment:
Hmm, not sure. I guess this really depends on how we define simple builds. I'm fine with only having triggers for prod builds, if others agree.